doc/source/serve/deployment-graph/http_endpoint_for_dag_graph.md

```diff
@@ -0,0 +1,29 @@
+# Pattern: Http endpoint for dag graph
+
+The example shows how to add endpoint for dag graph
+
+## Code
+
++++
+
+```{eval-rst}
+.. literalinclude:: ../doc_code/deployment_graph_dag_http.py
+   :language: python
+```
+
+````{note}
```

Comment:
hmm i think this should be three ` like ``` ?

Reply:
if you have nested, ```` is the way to indicate this is the outter section. (lmk if you know a better solution)

Reply:
oh you're nesting them, let me check again on the built html once your latest commit finishes